packages/etherlime-lib/deployed-contract/deployed-contract-wrapper.js

```diff
@@ -26,6 +27,11 @@ class DeployedContractWrapper {
 		this.estimate = this.contract.estimate
 		this.functions = this.contract.functions
 		this.filters = this.contract.filters
+		this.utils = {
+            getBalance: async function() {
```

Comment:
I think it is better to create the utils in another function and call it here. This way you will have clear readability of the functions.